lang/wren-cli/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem        1.0
+PortGroup         github 1.0
+PortGroup         makefile 1.0
+
+github.setup      wren-lang wren-cli 0.3.0
+
+categories        lang
+license           MIT
+platforms         darwin
+
+description       A command line tool for the Wren programming language
+
+long_description  The CLI project is a small and simple REPL and CLI tool for \
+                  running Wren scripts. It is backed by libuv to implement IO \
+                  functionality, and is a work in progress.
+
+homepage          https://wren.io/cli/
+
+maintainers       {gmail.com:herby.gillot @herbygillot} \
+                  openmaintainer
+
+checksums         rmd160  f92affe56d28b0697ec3e618e3b1fbfd515b2ccc \
+                  sha256  6ded2552c919f41b6e6c37801e1b2f29691de534c0d5328641d4daa6184cd850 \
+                  size    509880
+
+worksrcdir        ${worksrcdir}/projects/make.mac
+
+build.args-append verbose=1
+
+if {${os.arch} eq "i386"} {
+    build.args-append config=release_32bit
+} else {
+    build.args-append config=release_64bit
+}
```

Comment:
This looks like you're assuming Intel only. If so, I recommend adding `supported_archs x86_64 i386`. Or if these configurations only deal with bitness and don't care about architecture, then it should be `if {${os.arch} in "i386 ppc"}`.

Reply:
Changes made